CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 
 #### UWP
 
-* **[Fix]** Don't start a new session from logs in the background.
+* **[Fix]** Extend the current session instead of starting a new session when sending events from the background. Sessions are also no longer started in background by sending an event or a log from another service such as push, as a consequence the push registration information will be missing from crash events information.
```

Comment:
We don't have crash breadcrumbs for windows platforms. The message might be relevant in the future but for now the second part is confusing for UWP.